conduct.md

```diff
@@ -96,9 +96,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should not avoid reviewing documentation after a decision for the project has been made.
```

Comment:
I think line 101 could be a bit clearer for me. By "reviewing documentation" do you mean "updating"?

Reply:
Makes sense to me!

Reply:
Appreciate it guys! Thanks for the suggestions!